client/js/screens/Lessons/SurveyDifficulty/SurveyDifficultyContainer.js

```diff
@@ -0,0 +1,145 @@
+
+        <View
+          style={{
+            backgroundColor: "#F2C249",
+            width: 270,
+            height: 220,
+            justifyContent: "center",
+            alignItems: "center",
+            borderRadius: 10,
+            marginBottom: 20,
+            fontFamily: "Quicksand"
+          }}
+        >
+          <View
+            style={{
+              right: 0,
+              bottom: 0,
+              backgroundColor: "transparent",
+              borderStyle: "solid",
+              borderLeftWidth: 170,
+              borderBottomWidth: 210,
+              borderBottomColor: "transparent",
+              borderLeftColor: "#edcc74",
+              transform: [{ rotate: "180deg" }],
+              position: "absolute"
+            }}
+          />
+          <Image
+            style={{
+              width: 100,
+              height: 150
+            }}
+            source={require("../../../assets/images/illustrations/PNG/Tree_oranges.png")}
+          />
+          <TouchableOpacity
+            style={{
+              backgroundColor: "#25BDD8",
+              height: 35,
+              width: 130,
+              alignItems: "center",
+              justifyContent: "center",
+              borderRadius: 80
+            }}
+          >
+            <Text
+              style={{
+                color: "white"
+              }}
+            >
+              Beginner
+            </Text>
+          </TouchableOpacity>
+        </View>
+        <View
+          style={{
+            backgroundColor: "#59c9c9",
+            width: 270,
+            height: 220,
+            justifyContent: "center",
+            alignItems: "center",
+            borderRadius: 10
+          }}
+        >
+          <View
+            style={{
+              right: 0,
+              bottom: 0,
+              backgroundColor: "transparent",
+              borderStyle: "solid",
+              borderLeftWidth: 170,
+              borderBottomWidth: 210,
+              borderBottomColor: "transparent",
+              borderLeftColor: "#98CBD3",
+              transform: [{ rotate: "180deg" }],
+              position: "absolute"
+            }}
+          />
+          <Image
+            style={{
+              width: 100,
+              height: 150
+            }}
+            source={require("../../../assets/images/illustrations/PNG/Tree_oranges.png")}
+          />
+          <TouchableOpacity
+            style={{
+              backgroundColor: "#25BDD8",
+              height: 35,
+              width: 130,
+              alignItems: "center",
+              justifyContent: "center",
+              borderRadius: 80
+            }}
+          >
+            <Text
+              style={{
+                color: "white"
+              }}
+            >
+              Advanced
+            </Text>
+          </TouchableOpacity>
+        </View>
+
+        <View>
+          <TouchableOpacity>
+            <Text
+              style={{
```

Comment:
Put it into styles file

Reply:
I didn't rebase, it is already in develop